homeassistant/components/screenlogic/__init__.py

```diff
@@ -219,6 +219,7 @@ def device_info(self) -> DeviceInfo:
             equipment_model = f"Unknown Model C:{controller_type} H:{hardware_type}"
         return DeviceInfo(
             connections={(dr.CONNECTION_NETWORK_MAC, self.mac)},
+            identifiers={(DOMAIN, self.mac)},
```

Comment:
We don’t need to set identifiers if connections are set unless you need to use via_device

Reply:
Was testing each component as it's own device, but backed off of it. Forgot to remove this.

Reply:
[063d0d7](https://github.com/home-assistant/core/pull/85725/commits/063d0d7a5262eb3df8731b8a01d7274e902a39be)